operations/evaluate.go

```diff
@@ -40,7 +41,9 @@ func Evaluate() cli.Command {
 			}
 
 			p := &model.Project{}
-			_, err = model.LoadProjectInto(configBytes, "", p)
+			ctx := context.Background()
+			opts := model.GetProjectOpts{}
+			_, err = model.LoadProjectInto(ctx, configBytes, opts, "", p)
```

Comment:
We won't have to handle this right now, but in the future we're going to maybe need to add something to Opts to indicate what kind of thing we're looking at (mainline, patch, local) so that we know how to handle includes (for example here, we're going to need to use ioutil.ReadFile to handle includes. We may want to create an explicit ticket for it.